### PR TITLE
Implement complex sgn

### PR DESCRIPTION
**Pull request by rotu**

### CODE REVIEW

**Comment by ringabout** [comment]
Wrong branch, contribute to the devel branch please.

File: lib/pure/complex.nim [hunk 1]
```diff
@@ -81,6 +81,13 @@ func abs2*[T](z: Complex[T]): T =
   ## that is the squared distance from (0, 0) to `z`.
   ## This is more efficient than `abs(z) ^ 2`.
   result = z.re * z.re + z.im * z.im
+  
+func sgn*[T](z: Complex[T]): Complex[T] =
```
Comment (Varriount) [inline lib/pure/complex.nim:85]: Why not `sign`?
```suggestion
func sign*[T](z: Complex[T]): Complex[T] =
```

Comment (ghost) [reply lib/pure/complex.nim:85]: @Varriount because `sgn` is more common, it'll be more consistent, e.g. https://nim-lang.org/docs/math.html#sgn%2CT

Comment (Varriount) [reply lib/pure/complex.nim:85]: Oh well. I'd prefer for them to both be `sign`, but I see your point.

**Pull request merged**